src/test/groovy/net/fabricmc/loom/test/unit/LoomMocks.groovy

```diff
@@ -34,12 +37,14 @@ class LoomMocks {
     static IntermediaryMappingsProvider intermediaryMappingsProviderMock(String minecraftVersion, String intermediaryUrl) {
         def minecraftVersionProperty = GradleTestUtil.mockProperty(minecraftVersion)
         def intermediaryUrlProperty = GradleTestUtil.mockProperty(intermediaryUrl)
+		def downloaderProperty = GradleTestUtil.mockProperty(Download.&create as Function)
```

Comment:
There's something weird with the indents in this method (mixed tabs and spaces?)

Reply:
Humm, looks like we dont check this, will reformat all the groovy to use tabs separately.